### PR TITLE
Added windows script for nmos-testing tool

### PR DESCRIPTION
**Pull request by PallesenJulius**
Wanted to try the NMOS-testing tool mDNS discovery outside of the VM / WSL. Still didn't work :(

But this was a biproduct of the playing around and i thought it might be useful in the future 👍

The change in "check_internal_requirements" is a funny one. For whatever reason AMWA wrote their own requirements check? But its broken for windows, probably some case sensitivity or windows packages being differently named or something? 😂😂

### CODE REVIEW

File: nmostesting/NMOSTesting.py [hunk 1]
```diff
@@ -975,6 +975,11 @@ def get_package_name(requirement):
 
 
 def check_internal_requirements():
+    # This manual requirement check doesn't seem to work for Windows. Just return early on Windows.
+    # No need to bother to understand what's going on...
+    if platform.system() == "Windows":
+        return
```
Comment (SergejAnZu) [inline nmostesting/NMOSTesting.py:982]: Add log output that check is skupped on Windows?

File: run_nmos_testing.ps1 [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Windows script to start nmos-testing tool
+
+param(
+    [switch]$e,  # Default is $false
+    [switch]$u,  # Default is $false
+    [string]$c = "../../../BuzzVirtual-Gen2-documentation/certificates/"  # Default certificate path
```
Comment (SergejAnZu) [inline run_nmos_testing.ps1]: Can we use REPO_ROOT approach as in the original? CERT_DIR="${REPO_ROOT}/../BuzzVirtual-Gen2-documentation/certificates"

File: run_nmos_testing.ps1 [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+# limitations under the License.
+
+from . import Config as CONFIG
+
+# Copy this file to "UserConfig.py" to change configuration values.
+
+CONFIG.CERT_CLIENT = ("$c/mgmt-client-combined.pem.crt", "$c/mgmt-client.pem.key")
+CONFIG.CERT_TRUST_ROOT_CA = False
+CONFIG.ENABLE_HTTPS = $e
+CONFIG.DNS_SD_MODE = "$dns_mode"
```
Comment (SergejAnZu) [inline run_nmos_testing.ps1]: Do we need to reflect this in original launch?

Comment (PallesenJulius) [reply run_nmos_testing.ps1]: Not that i would know. Do we ever test with unicast?

File: run_nmos_testing.ps1 (hunk 2 above)
Comment (PallesenJulius) [reply run_nmos_testing.ps1]: Oh yea that was missing all over the place, good catch.

File: nmostesting/NMOSTesting.py (hunk 1 above)
Comment (PallesenJulius) [reply nmostesting/NMOSTesting.py:982]: Don't know why they manually check requirements anyways, but sure 😂

File: run_nmos_testing.ps1 (hunk 3 above)
Comment (SergejAnZu) [reply run_nmos_testing.ps1]: Afaik we don't

**Pull request merged**